src/components/Article/Article.jsx

```diff
@@ -1,5 +1,69 @@
 // import a css file containig article styles
-
+import './Article.css';
 // Create an Article function returning the HTML of article block
+function Article() {
+  return (
+    <article className="article">
+      <h1 className="article__title">Headline</h1>
```

Comment:
if you just return something without logic - you can do it like that
```suggestion
function Article() (
    <article className="article">
      <h1 className="article__title">Headline</h1>
```